### DISCUSSION TITLE
using github account showing My account is disabled.

### DISCUSSION THREAD

**Discussion by sachingllfl** [vercel/community#3806] [Account Issue] [answered]
### Summary

![image](https://github.com/vercel/community/assets/143379127/72eb41ae-6d89-4c9e-abc2-ea4d28eb08d2)

Showing This user account is blocked. while login with GitHub account

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/143379127/72eb41ae-6d89-4c9e-abc2-ea4d28eb08d2)

Showing This user account is blocked. while login with GitHub account

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy.
